x-pack/test/tsconfig.json

```diff
@@ -140,7 +140,6 @@
     "@kbn/securitysolution-endpoint-exceptions-common",
     "@kbn/osquery-plugin",
     "@kbn/cases-api-integration-test-plugin",
-    "@kbn/security-solution-plugin/public/management/cypress",
```

Comment:
Hopefully it was the reason for circular dependencies when relocating DA platform tests